gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/rest/api/service/impl/ApiService_RollbackTest.java

```diff
@@ -77,4 +82,27 @@ public void rollback_should_import_api_using_apiduplicator_service() throws Json
         verify(apiDuplicatorService, times(1))
             .updateWithImportedDefinition(GraviteeContext.getExecutionContext(), "my-api-id", "my-serialized-api");
     }
+
+    @Test
+    public void rollback_should_rollback_emulation_engine_flag() throws JsonProcessingException {
+        RollbackApiEntity rollbackApiEntity = new RollbackApiEntity();
+        rollbackApiEntity.setId("idtest");
+        rollbackApiEntity.setName("nametest");
+        rollbackApiEntity.setExecutionMode(ExecutionMode.V3);
+        rollbackApiEntity.setDescription("descriptiontest");
+        String mapperString = "{\"id\":\"idtest\",\"name\":\"nametest\",\"description\":\"descriptiontest\"}";
+        when(objectMapper.writeValueAsString(rollbackApiEntity)).thenReturn(mapperString);
```

Comment:
Please remove this mock, since it is one of the parameter you want to check line 103.